components/trezor_bridge/resources/trezor/data/bridge/releases.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
Since the entire data/ directory is copied from the published `trezor-connect` npm module, it is already available to the source code via `node_modules/trezor-connect/data`. Therefore we do not need to commit this large amount of data to git.
Instead, you can use GN `copy` template to copy each file before packaging using the same grit .grd file, e.g.

```gn
data_dir = "//brave/node_modules/trezor-connect/data"

copy("data") {
  visibility = [ ":*" ]
  sources = [
    "$data_dir/bridge/latest.txt",
    "$data_dir/bridge/releases.json",
    "$data_dir/bridge/coins.json",
    "$data_dir/bridge/config.json",
    ...
    "$data_dir/bridge/messages/messagesv7.json",
    ...
  ]
  outputs = [ "$root_gen_dir/components/trezor_bridge/resources/trezor/data/{{source_file_part}}" ]
}
```

Reply:
done